### DISCUSSION TITLE
Window Controls Overlay API Support

### DISCUSSION THREAD

**Discussion by jsormaz** [quasarframework/quasar#16149] [CLI - PWA mode] [answered]
I tried for a short while to get this to work, with no success. Is there something inherent in quasar that does not support it?

I couldn't even get the PWA to take the 'window-controls-overlay' display override value.

I tried both on a dev (https enabled) as well as full build.

https://developer.mozilla.org/en-US/docs/Web/API/Window_Controls_Overlay_API

**Comment by metalsadman** [comment]
it's an experimental feature

**Comment by rstoenescu** [comment, marked as answer]
There is nothing specific in regards to Quasar. If the browser API is supported, then it should work. Check the support table on MDN for it and make sure that you have the specified minimum browser version. Also note that it is marked as experimental and the API might change.
